spec/service_consumers/pact_helper.rb

```diff
@@ -209,4 +209,22 @@
       FactoryGirl.create(:draft_content_item, title: 'Content Item C', base_path: '/yet-another-base-path', format: 'topic', publishing_app: 'whitehall')
     end
   end
+
+  provider_state "there are two documents that link to the wantend document" do
```

Comment:
Makes more sense to me. I think specifics are also worth including, but maybe quote them to show that they're data points.